logic_mod/Blocks/ExtAltimeterBlock.cs

```diff
@@ -96,6 +96,17 @@ protected override void Awake()
             heightSlider.DisplayName = ">" + heightSlider.DisplayName;
         }
 
+        public Dictionary<string, object> GetPos()
+        {
+            var myPos = Rigidbody.position;
+            return new Dictionary<string, object>
+            {
+                { "x", myPos.x },
+                { "y", myPos.y },
+                { "z", myPos.z }
+            };

```

Comment:
The helper function converting Vector3->Dictionary, made for anglometer, also should be used here